### ISSUE TITLE
Find a way to properly integrate viz from blocks d3 in posts

### ISSUE THREAD

**Issue by clemsos** [theatlasofdata/theatlasofdata.earth#26]
the iframe currently fails

```html 
<iframe width="600" height="600" marginwidth="0" marginheight="0" src="http://bl.ocks.org/WebTrackingCartography/raw/e59cfc5870d6ec8990a30e05fac72f74/" />
```

**Comment by MAKIO135** [comment]
bl.ocks.org prevents being embedded:
> Refused to display 'https://bl.ocks.org/WebTrackingCartography/raw/f2bca61a0780f47dca5f618700d76065/' in a frame because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'self' *.ocks.org".

But from [bl.ocks About page](https://bl.ocks.org/-/about):
> Bl.ocks (pronounced “Blocks”) is a simple viewer for sharing code examples hosted on GitHub Gist. 

So, instead of using bl.ocks, viz could loaded via [Github gists API](https://developer.github.com/v3/gists/)

**Comment by clemsos** [comment]
@MAKIO135 Can we use rawgit?

**Comment by MAKIO135** [comment]
Not sure. Need to understand how bl.ocks.org works.

**Comment by MAKIO135** [comment]
https://github.com/mbostock/bl.ocks.org for reference

**Comment by MAKIO135** [comment]
Accessing a public gist can be done anonymously via a simple request

**Comment by clemsos** [comment]
@MAKIO135 So we just fetch all files from gist then figure out how to parse and display them based on their extension?

**Comment by clemsos** [comment]
@clemsos Maybe we should just host everything ourselves...  Will be much safer for the project anyway. 

Actually we should think about some dataviz graphic rules that we apply to all viz

**Comment by clemsos** [comment]
finally I made a lib to backup things locally from gist https://www.npmjs.com/package/blockfetcher

**Comment by clemsos** [comment]
solved by dc509a8. we now have all files backed up in the repo.
